### DISCUSSION TITLE
Remove: Portugal Channels

### DISCUSSION THREAD

**Discussion by spotanjo3** [iptv-org/community#235] [Ideas]
I dont think it have nothing to do with DMCA but... Those are not needed:

1001 Noites (720p)(not 24/7)
Locomotion TV (480p)(not 24/7)
RTP 1,2,3, (not 24/7)
Sobrenatural (720p)(not 24/7)
RTP AFRICA (not 24/7)

Just want to make it clean and properly like a real TV. :)

Thank you kindly.

**Comment by BellezaEmporium** [comment]
For which reasons should we delete those streams ? They're Portuguese channels, even if they're not streamed on ISPs.

Some also are alternate streams, in case the main link gets off.

**Comment by spotanjo3** [comment]
Explain to me what is "not 24/7" then ? ^_^'

**Comment by BellezaEmporium** [comment]
The channels marked Not 24/7 is either because the servers that host them can have some outages and/or they do not stream all the time, hence we add this sign next to the channel name to prevent the bot thinking that it's an offline channel that must be deleted.

**Comment by spotanjo3** [comment]
I understand. Is there possibly to add RTP 1,2,3 as other servers without a "not 24/7" ? Curiosity. ^_^

**Comment by BellezaEmporium** [comment]
Won't be easy, since the Not 24/7 channels... are the official RTP streams.
